grommet-leaflet/src/themes/base.js

```diff
@@ -79,6 +79,41 @@ const base = {
     pad: 'small',
     round: 'xsmall',
   },
+  // tiles: {
+  // Protomaps [basemap layers](https://docs.protomaps.com/basemaps/layers#basemap-layers) derived from OpenStreetMap.
```

Comment:
General question, how were these colors selected?

Also, should we be considering what a light/dark theme structure would look like before we commit to this theme structure?

Reply:
Added TODO comment